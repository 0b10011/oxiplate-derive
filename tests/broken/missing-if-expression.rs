use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline = "{% if %}"]
struct Data {}

fn main() {
    let data = Data {};

    print!("{}", data);
}
