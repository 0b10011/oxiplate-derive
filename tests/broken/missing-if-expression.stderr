error: Expected an expression after `if`
 --> tests/broken/missing-if-expression.rs:4:28
  |
4 | #[oxiplate_inline = "{% if %}"]
  |                            ^
